templates/staff/project/create.html

```diff
@@ -34,7 +34,7 @@
       <div class="flex flex-col items-stretch gap-y-6 w-full max-w-3xl">
         {% #form_fieldset %}
           {% form_legend text="Select an org" %}
-          {% multiselect class="w-full" data-max-items=1 field=form.fields.org multiple=False input=form.org %}
+          {% multiselect class="w-full" data-max-items=1 field=form.fields.orgs multiple=False input=form.orgs %}
```

Comment:
Based on the arguments alone, I think this multiselect allows a user to select one item. Don't we want a user to select one or more items?

Reply:
No!  This, and #3571, are all about changing the schema to support multiple orgs but not actually supporting them in the UI, so in practice we still only have one Org per Project, but when we're ready to support them there won't be a mammoth change needed to do so.